apps/ligretto-frontend/src/components/blocks/game/Card/Card.tsx

```diff
@@ -130,6 +130,8 @@ const StyledCardValue = styled(Typography)(() => ({
   fontSize: 'inherit',
 }))
 
+export const StubCard: React.FC = () => <StyledCard size="medium" color={CardColors.empty} />
```

Comment:
Why do you need it?

Reply:
UPD: Removed `StubCard`, fixed `CardHotkeyBadge` posiiton via `alignItems="flex-start"` at `<Stack/>`

![image](https://user-images.githubusercontent.com/44056222/198898032-02b9e72b-4f8d-49a0-9a91-244b6dab5094.png)